boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/IAMStarter.java

```diff
@@ -113,14 +121,19 @@ private static String streamOutput(InputStream inputStream) throws ExecutionExce
         // Stream microservice output
         ExecutorService executor = Executors.newSingleThreadExecutor();
 
+        String readyMarker = config.getString(IAMConfiguration.READY_MARKER);
+        String errorMarker = config.getString(IAMConfiguration.ERROR_MARKER);
+
         Future<String> future = executor.submit((Callable) () -> {
 
             try (BufferedReader br = new BufferedReader(new InputStreamReader(inputStream))) {
                 String line;
 
                 while ((line = br.readLine()) != null) {
-                    if (line.contains(config.getString(IAMConfiguration.READY_MARKER))) {
+                    if (line.contains(readyMarker)) {
                         break;
+                    } else if (line.contains(errorMarker)) {
```

Comment:
Maybe you could also log the error before throwing the exception?

Reply:
I can only log the read line. I tried it and it gives no information to the user. I deem that it is cleaner to do that way. The user can see the error details in IAM logs.